### DISCUSSION TITLE
830F463D-2F5B-5630-47C8-A96C7EB73362

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#211] [General]
# M 1 (Crab Nebula)

The Crab Nebula (also known as Messier 1, NGC 1952, Taurus A) is a supernova remnant and pulsar wind nebula in the constellation of Taurus. The nebula was discovered by English astronomer John Bevis in 1731, and it corresponds with a bright supernova recorded by Chinese astronomers in 1054, making it the first astronomical object identified that corresponds with a historical supernova explosion. It is located around 6,500 away from the Solar System

https://celestia.mobi/addon?item=830F463D-2F5B-5630-47C8-A96C7EB73362

**Comment by dave-kaye** [comment]
Starting 6/25, Celestia 1.7.0 has supported nebula types.

To use this feature, add the following two lines to the first data grouping in the file crab_nebula.dsc

    Type    "SupernovaRemnant"
    AbsMag  -3.1

Data from Wikipedia - Crab Nebula

**Comment by LangLangFCS** [comment]
This plugin is really low-quality 
(눈_눈)
